addons/web/static/src/core/datetime/datetime_picker.scss

```
@@ -138,7 +138,7 @@
     // Compute  spacing  between the two month by compensating buttons size and tables gap
     
     .o_zoom_out {
-        gap: o-to-rem($btn-border-width) * 4 + o-to-rem($o-font-size-base) * 2 + o-to-rem($btn-padding-x * 4);
+        gap: calc(#{$btn-border-width * 4} +  #{$o-font-size-base * 2} +  #{$btn-padding-x * 4});
```

I don't think it's necessary to use `calc` in this case. I think the main issue comes from adding values to `o-to-rem` that are already in `rem`, which divides them by 16 and creates the gap problem.
Could you try this?
```suggestion
        gap: o-to-rem($btn-border-width) * 4 + $o-font-size-base * 2 + $btn-padding-x * 4;
```

---

That doesn't work either, we already tried to find a workaround with converting only the first value to rem (I still pushed on runbot so you can check). IMO we should go for a calc if it avoids all these potential issues